robottelo/constants/repos.py

```diff
@@ -1,6 +1,6 @@
 """Only External Repos url specific constants module"""
 
-
+PULP_FIXTURE_ROOT = 'https://fixtures.pulpproject.org/'
```

Comment:
This is more of your team's decision, but I recommend moving these fixtures to our hosted mirror with content - https://github.com/SatelliteQE/fedorapeople-repos. Not sure how relevant this suggestion is relative to what this test is trying to do, but my concerns are possible network issues and external service availability.